NewEvaluation.css

```diff
@@ -180,6 +148,52 @@ h3{
 
 }
 
+/*---------------------- TextArea Box---------------------------*/
+
+.user-textarea textarea{
+	width:100%;
+	height: 100px;
+}
+
+
+
+/*------------------- Legend Box ------------------*/
+.legend-box legend{
+	color:#03a8f3;
+	font-size: 18px;
+	font-weight: 500;
+	padding-left: 1em;
+	padding-right: 1em;
+}
+
+.legend-box li{
+	float:left;
+	width: 33%;
```

Comment:
I noticed initially that the tables is divided in 3 columns, there is where  got I 33% from. This 33% would be for a pair of label and dropdown menu so now we have the parent that is 1/3 of the width of the parent and has 2 elements that should be half each. Initial when I made the code I didn't check the initial values of paddings margins and borders of the elements and as a go-around method I gave them some aproximation values of 40% and 49% that would align them in the column the way I wanted